<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>820</width>
    <height>590</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QtMaterialTabs" name="tabwidget" native="true">
    <property name="geometry">
     <rect>
      <x>-10</x>
      <y>-20</y>
      <width>841</width>
      <height>80</height>
     </rect>
    </property>
   </widget>
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>40</y>
      <width>821</width>
      <height>541</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="page_jiankangjiance">
     <widget class="QtMaterialCheckBox" name="checkBox_allanpei">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>450</y>
        <width>141</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 300 11pt &quot;Ubuntu&quot;;</string>
      </property>
      <property name="text">
       <string>All_Current</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_rov_cabin">
      <property name="geometry">
       <rect>
        <x>480</x>
        <y>40</y>
        <width>101</width>
        <height>81</height>
       </rect>
      </property>
      <property name="text">
       <string>rovCABIN</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_rov_em">
      <property name="geometry">
       <rect>
        <x>650</x>
        <y>180</y>
        <width>81</width>
        <height>81</height>
       </rect>
      </property>
      <property name="text">
       <string>rovEM</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_boat_em">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>180</y>
        <width>81</width>
        <height>81</height>
       </rect>
      </property>
      <property name="text">
       <string>boatEM</string>
      </property>
     </widget>
     <widget class="QWidget" name="horizontalLayoutWidget">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>440</y>
        <width>381</width>
        <height>41</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QtMaterialCheckBox" name="checkBox_wendu">
         <property name="text">
          <string>温度</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QtMaterialCheckBox" name="checkBox_shidu">
         <property name="text">
          <string>湿度</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_3">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QtMaterialCheckBox" name="checkBox_boatcabin">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>140</y>
        <width>141</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 300 11pt &quot;Ubuntu&quot;;</string>
      </property>
      <property name="text">
       <string>Boat_Cabin</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_boat_cabin">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>50</y>
        <width>158</width>
        <height>65</height>
       </rect>
      </property>
      <property name="text">
       <string>boatCABIN</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_dianji_anpei">
      <property name="geometry">
       <rect>
        <x>650</x>
        <y>320</y>
        <width>61</width>
        <height>121</height>
       </rect>
      </property>
      <property name="text">
       <string>dianjidianliu</string>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>70</y>
        <width>67</width>
        <height>17</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 255 11pt &quot;Ubuntu&quot;;</string>
      </property>
      <property name="text">
       <string>CONNECT</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_all_anpei">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>320</y>
        <width>61</width>
        <height>121</height>
       </rect>
      </property>
      <property name="text">
       <string>allANPEI</string>
      </property>
     </widget>
     <widget class="QChartView" name="widget_info" native="true">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>205</y>
        <width>381</width>
        <height>231</height>
       </rect>
      </property>
     </widget>
     <widget class="QtMaterialToggle" name="widget_switch" native="true">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>10</y>
        <width>71</width>
        <height>51</height>
       </rect>
      </property>
     </widget>
     <widget class="QtMaterialCheckBox" name="checkBox_rovcabin">
      <property name="geometry">
       <rect>
        <x>470</x>
        <y>140</y>
        <width>141</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 300 11pt &quot;Ubuntu&quot;;</string>
      </property>
      <property name="text">
       <string>Rov_Cabin</string>
      </property>
     </widget>
     <widget class="QtMaterialCheckBox" name="checkBox_dianjianpei">
      <property name="geometry">
       <rect>
        <x>640</x>
        <y>440</y>
        <width>141</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 300 11pt &quot;Ubuntu&quot;;</string>
      </property>
      <property name="text">
       <string>EM_Current</string>
      </property>
     </widget>
     <widget class="QtMaterialToggle" name="widget_boatem_station" native="true">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>270</y>
        <width>40</width>
        <height>45</height>
       </rect>
      </property>
     </widget>
     <widget class="QtMaterialToggle" name="widget_rovem_station" native="true">
      <property name="geometry">
       <rect>
        <x>670</x>
        <y>270</y>
        <width>40</width>
        <height>45</height>
       </rect>
      </property>
     </widget>
     <widget class="QtMaterialToggle" name="widget" native="true">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>135</y>
        <width>40</width>
        <height>45</height>
       </rect>
      </property>
     </widget>
     <widget class="QtMaterialToggle" name="widget_2" native="true">
      <property name="geometry">
       <rect>
        <x>580</x>
        <y>135</y>
        <width>40</width>
        <height>45</height>
       </rect>
      </property>
     </widget>
     <widget class="Line" name="line">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>75</y>
        <width>81</width>
        <height>20</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QtMaterialToggle" name="widget_3" native="true">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>60</y>
        <width>40</width>
        <height>45</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_shuizhijiance">
     <widget class="ProgressWater" name="widget_shuizhipinggu" native="true">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>90</y>
        <width>261</width>
        <height>191</height>
       </rect>
      </property>
     </widget>
     <widget class="QtMaterialFlatButton" name="pushButton_shuizhipinggu">
      <property name="geometry">
       <rect>
        <x>470</x>
        <y>340</y>
        <width>151</width>
        <height>71</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 300 13pt &quot;Ubuntu&quot;;</string>
      </property>
      <property name="text">
       <string>Assessment</string>
      </property>
     </widget>
     <widget class="QtMaterialProgress" name="progressBar">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>420</y>
        <width>261</width>
        <height>21</height>
       </rect>
      </property>
      <property name="value">
       <number>24</number>
      </property>
     </widget>
     <widget class="QTextEdit" name="textEdit">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>130</y>
        <width>251</width>
        <height>91</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-image: url(:/rec/whitebackpic.jpg);
font: 300 11pt &quot;Ubuntu&quot;;</string>
      </property>
      <property name="html">
       <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
hr { height: 1px; border-width: 0; }
li.unchecked::marker { content: &quot;\2610&quot;; }
li.checked::marker { content: &quot;\2612&quot;; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Ubuntu'; font-size:11pt; font-weight:300; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;######################&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;请点击Assessment按键进行评估&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;######################&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_wulianwang">
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <layout class="QGridLayout" name="gridLayout"/>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>820</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QtMaterialTabs</class>
   <extends>QWidget</extends>
   <header location="global">qtmaterialtabs.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>QtMaterialCheckBox</class>
   <extends>QCheckBox</extends>
   <header location="global">qtmaterialcheckbox.h</header>
  </customwidget>
  <customwidget>
   <class>QChartView</class>
   <extends>QWidget</extends>
   <header location="global">qchartview.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>QtMaterialToggle</class>
   <extends>QWidget</extends>
   <header location="global">qtmaterialtoggle.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>QtMaterialFlatButton</class>
   <extends>QPushButton</extends>
   <header location="global">qtmaterialflatbutton.h</header>
  </customwidget>
  <customwidget>
   <class>ProgressWater</class>
   <extends>QWidget</extends>
   <header location="global">progresswater.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>QtMaterialProgress</class>
   <extends>QProgressBar</extends>
   <header location="global">qtmaterialprogress.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
